src/shared/interfaces/categorized-validation-errors.ts

```diff
@@ -0,0 +1,6 @@
+import { SchemaValidationErrors } from './schema-validation-errors';
+
+export interface CategorizedValidationErrors {
```

Comment:
@jmartinm  Actually `ValidationMessages` interface is not used is a leftover from a previous version of this PR. So, it will not be confusing but do we want to change it anyway? I think is more consistent to  leave it as errors because all our internal maps are called like this independently if there are errors or warnings inside.